frontend/src/components/table/Table.js

```diff
@@ -452,16 +452,12 @@ export default class Table extends PureComponent {
   renderEmptyInfo = (rows) => {
     const { emptyText, emptyHint, pending } = this.props;
 
-    if (pending) {
-      return false;
-    }
-
     if (!rows.length) {
       return (
```

Comment:
check the pending flag here and if true then return null

Reply:
i mean, the whole "div className="empty-info-text"" div shall go out in case of pending=true

Reply:
or?

Reply:
there is the case when no data returned ( no rows). If we do not display the 'empty-info-text'  div we will screw the way the spinner is shown for that specific case. So it has to be like that.

Reply:
pls document that in the code.... it's not obvious